recipe/meta.yaml

```diff
@@ -2,6 +2,7 @@
 {% set name = "implicit" %}
 {% set version = "0.7.0" %}
 {% set sha256 = "e7bcf0c267404f0e579f268515174e981996bb268106c5be869d312bf48ab72e" %}
+{% set number = "4" %}
 
 {% set implicit_proc_type = "cpu" if cuda_compiler_version == "None" else "gpu" %}  # [linux64]
```

Comment:
Do we need to change these `linux64` selectors to just `linux` for aarch64?  (since afaict linux64 is x86_64 linux only).

Reply:
Thanks Ben! 🙏

Yes we should change the logic here for `linux_aarch64`. Have rewritten it to be more general such that `linux` selectors are unneeded

This would be useful if Windows + CUDA builds became viable here in the future